### PR TITLE
Remove dead resource strings from Microsoft.VisualBasic.Core

### LINKED ISSUE #40016: Some dead resource strings in Microsoft.VisualBasic.Core
Several area not referenced in code, eg., EnvVarNotFound_Name, WinForms_RecursiveFormCreate, WinForms_SeeInnerException...

These can be removed from src\libraries\Microsoft.VisualBasic.Core\src\Resources\Strings.resx

### PR DESCRIPTION
**Pull request by Marusyk**
Fixes #40016

Please review
Thank you in advance

### CODE REVIEW

**Comment by jkotas** [comment]
```
  Starting:    Microsoft.VisualBasic.Core.Tests (parallel test collections = on, max threads = 2)
    Microsoft.VisualBasic.Tests.ConversionTests.ErrorToString_WithErrorNumber(errorNumber: 7) [FAIL]
      Assert.NotNull() Failure
      Stack Trace:
        /_/src/libraries/Microsoft.VisualBasic.Core/tests/ConversionTests.cs(145,0): at Microsoft.VisualBasic.Tests.ConversionTests.ErrorToString_WithErrorNumber(Int32 errorNumber)
    Microsoft.VisualBasic.Tests.ErrObjectTests.FilterDefaultMessage [FAIL]
      Assert.NotEqual() Failure
      Expected: Not "Exception from HRESULT: 0x80"
      Actual:   "Exception from HRESULT: 0x80"
      Stack Trace:
        /_/src/libraries/Microsoft.VisualBasic.Core/tests/ErrObjectTests.cs(91,0): at Microsoft.VisualBasic.Tests.ErrObjectTests.FilterDefaultMessage()
```

File: src/libraries/Microsoft.VisualBasic.Core/src/Resources/Strings.resx [hunk 1]
```diff
@@ -116,285 +116,15 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
-  <data name="ID3" xml:space="preserve">
-    <value>This Error number is obsolete and no longer used.</value>
-  </data>
-  <data name="ID5" xml:space="preserve">
-    <value>Procedure call or argument is not valid.</value>
-  </data>
-  <data name="ID6" xml:space="preserve">
```
Comment (cston) [inline src/libraries/Microsoft.VisualBasic.Core/src/Resources/Strings.resx:125]: >ID6 [](start = 14, length = 3)

It looks like a number of the "ID#" resources may be used for exception messages from `ErrObject`. For instance ID6, ID9, ID11, ID13, ...

See `ErrObject.MapExceptionToNumber()` and `ErrObject.FilterDefaultMessage()`.

Comment (Marusyk) [reply src/libraries/Microsoft.VisualBasic.Core/src/Resources/Strings.resx:125]: Thanks, I will revert all IDs

**Comment by jkotas** [comment]
Opened https://github.com/dotnet/runtime/issues/40115 on the test failure

**Review by jkotas — APPROVED** [review]
Thank you!

**Comment by danmosemsft** [comment]
Nice cleanup, thank you @Marusyk . If you are interested in another contribution .. https://github.com/dotnet/runtime/issues?q=is%3Aopen+is%3Aissue+label%3Aup-for-grabs

**Pull request merged**